modules/launcher/src/main/java/org/mule/module/launcher/artifact/AbstractArtifactClassLoader.java

```diff
@@ -24,12 +27,18 @@
  */
 public abstract class AbstractArtifactClassLoader extends FineGrainedControlClassLoader implements ArtifactClassLoader
 {
+    private static final String DEFAULT_LEAK_CLEANER_CLASS = "DefaultLeakCleaner.class";
```

Comment:
I tried to use it, but it didn't work. Here the class is just a resource, that's why I need to put the suffix .class.